db/warehouse/migrate/20240731144633_add_verified_by_project_id_to_current_living_situation.rb

```diff
@@ -0,0 +1,6 @@
+class AddVerifiedByProjectIdToCurrentLivingSituation < ActiveRecord::Migration[7.0]
+  def change
+    add_column :CurrentLivingSituation, :verified_by_project_id, :integer
+    add_foreign_key :CurrentLivingSituation, :Project, column: :verified_by_project_id, validate: false
```

Comment:
This can't be a foreign key, can it?  I guess if it can be null, then that might be ok, but we won't have this for any non CE project (right?) and won't have it from imported data.

Reply:
It's a nullable foreign key, from my understanding that should be fine?